src/lib/tradesHandler.ts

```diff
@@ -274,9 +230,15 @@ async function handleStopLossOrder(
 
 const handleTakePartialProfitOrder = async (trade: ExtendedTradesDataType) => {
   try {
+    // sell ~50%, ceiled value to prevent fractional trades.
```

Comment:
Maybe skip the 50% number here and mention the config var instead?

Reply:
👍